### DISCUSSION TITLE
Investigating antennas numbers: Unexpected results in Sionna RTX tutorial and code implementation

### DISCUSSION THREAD

**Discussion by yacine074** [NVlabs/sionna#161] [General]
In the published tutorial on [Sionna RTX](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Introduction.html), a single antenna was used for both the transmitter and receiver. When calculating the frequency response of the channel, the shape of h_freq was (1, 1, 2, 1, 1, 14, 48), indicating the presence of two receiving antennas (num_rx_ant = 2). The question arises as to why there are two receiving antennas instead of just one in this scenario.

In my [code](https://colab.research.google.com/drive/158kLUkRZPj11S7kMA9Q4Z8QqQq26kYMX?usp=sharing), I have simulated a single transmit antenna and a single receiver antenna with the corresponding different positions, with the goal of calculating the frequency responses of the channels. However, I am encountering a discrepancy where the number of transmitter antennas is being reported as 2 and the number of receiver antennas is also being reported as 2, and not 1, despite explicitly specifying a single antenna using a panel array with 1 column and a single row.

I tried using sionna.rt.Antenna, but it doesn't seem to work ("TypeError: array must be an instance of AntennaArray").

**Comment by faycalaa** [comment]
Hi,

In the tutorial you are referring to, a dual-polarized antenna is used for the receiver (```polarization="cross")```, leading to two antennas (one for each polarization direction).

Similarly, in your code, you have set a cross-polarized antenna for both the transmitter and receiver, which explains the observed number of antennas.

**Comment by yacine074** [comment]
Hello, 

Thank you so much for your very clear answer.
